pkg/auth/model/model.go

```diff
@@ -111,96 +126,125 @@ func (s *Statements) Scan(src interface{}) error {
 	return json.Unmarshal(data, s)
 }
 
-type Credential struct {
-	AccessKeyID                   string    `db:"access_key_id"`
-	SecretAccessKey               string    `db:"-" json:"-"`
-	SecretAccessKeyEncryptedBytes []byte    `db:"secret_access_key" json:"-"`
-	IssuedDate                    time.Time `db:"issued_date"`
-	UserID                        int64     `db:"user_id"`
-}
+// authPrefix - key prefix in the KV model
+const authPrefix = "auth"
 
-// For JSON serialization:
-type CredentialKeys struct {
-	AccessKeyID     string `json:"access_key_id"`
-	SecretAccessKey string `json:"secret_access_key"`
+//TODO - delete no lint comments when function is used
+//nolint
+func kvUserPath(userName string) string {
+	return kv.FormatPath(authPrefix, "users", userName)
 }
 
+//nolint
 func userFromProto(pb *UserData) *User {
 	return &User{
-		ID:                pb.ID,
+		ID:                int64(binary.LittleEndian.Uint64(pb.Id)),
```

Comment:
About this solution:
I'm not sure it works properly, and it unfortunately does not solve the problem I've mentioned before with the overflow.
I think the solution required here will have to include a schema change which will modify ID column to string / []byte and generate new IDs for all existing entities

Reply:
but this will happen in the migration

Reply:
For this to happen in the migration we need to create new structs for the KV models which will be used after the migration. Currently we are using the same structs which have the 'int' type for the ID fields - so the migration doesn't really help us here. What we need is to either do that or modify the current structs which requires DB schema version migration